src/sdk.ts

```diff
@@ -86,12 +98,24 @@ export class FuseSDK {
    */
   async executeBatch(
     calls: Array<ICall>,
-    txOptions?: typeof Variables.DEFAULT_TX_OPTIONS
+    isIndependentTransaction?: boolean,
+    txOptions?: typeof Variables.DEFAULT_TX_OPTIONS,
   ): Promise<ISendUserOperationResponse | null | undefined> {
     txOptions = txOptions ?? Variables.DEFAULT_TX_OPTIONS;
     const initialFees = BigInt(txOptions.feePerGas);
     this.setWalletFees(initialFees);
 
+    if(isIndependentTransaction) {
```

Comment:
Can you please expain how and when this flag should be used?

Reply:
@leonprou when dev need to execute multiple batches, they can pass this flag to the executeBatch function.

Reply:
Ok I see,
I have concerns about the re-initialization of the wallet. It creates a new client so if we use await with the previous client we are loosing the reference.